src/02.Components/CheckedList.js

```diff
@@ -37,19 +33,21 @@ export default function List({
                 onInputChange={handleInputChange}
                 onToggleCheckbox={() => toggleIsChecked(id)}
                 onDelete={() => deleteListItem(id)}
-                onEnter={() => addListItemOnEnter(id)}
+                onEnter={handelEnter}
               />
             )
           })}
         </ListStyled>
       )}
     </>
   )
+  function handelEnter() {}
```

Comment:
```suggestion
  function handleEnter() {}
```
This function is empty, you should fill or delete it before merging :)